index.ts

```diff
@@ -387,7 +390,7 @@ class DatabaseFake {
     return {
       page,
       isDone,
-      continueCursor,
+      continueCursor: continueCursor!,
```

Comment:
Is it possible there won't be any result & it'll still be null?

Reply:
nope it's an infinite loop with two breaks and we set the variable before each break